database/worker/index_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/DATA-DOG/go-sqlmock"
 )
 
-func TestWorker_Engine_CreateWorkerIndexes(t *testing.T) {
+func Indexes(t *testing.T) {
```

Comment:
Can we keep the testing name structure like we do [here](https://github.com/go-vela/server/blob/e06d557dd12cb1c6477b7403d7815f091cf25e51/database/build/index_test.go#L13), [here](https://github.com/go-vela/server/blob/e06d557dd12cb1c6477b7403d7815f091cf25e51/database/repo/index_test.go#L13), and [here](https://github.com/go-vela/server/blob/e06d557dd12cb1c6477b7403d7815f091cf25e51/database/pipeline/index_test.go#L13)?